### PR TITLE
Add r-base container

### PR DESCRIPTION
**Pull request by shaoyuta**

### CODE REVIEW

File: R/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+modify the container images.
+
+1. Clone the clearlinux/dockerfiles repository.
+    ```
+    git clone https://github.com/clearlinux/dockerfiles.git
+    ```
+
+2. Change to the directory of the application:
+    ```
+    cd R/
```
Comment (qzheng527) [inline R/README.md]: Usually we name the folder with the same name of  the container. 
I suggest you use "r-base" for the folder name as well.

File: R/hooks/post_push [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+. ../docker-hooks.sh
+
+image="clearlinux/r-base"
+package=r-base
```
Comment (qzheng527) [inline R/hooks/post_push]: The package you are tracking is not "r-base".
There is no "r-base" package in clear linux.
It should be "R".

**Comment by shaoyuta** [comment]
Thanks! I have updated this commit.

File: r-base/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+1. Pull the image from Docker Hub: 
+    ```
+    docker pull clearlinux/r-base
+    ```
+
+2. Start a container using the examples below:
+
+    ```
+    docker run -d clearlinux/r-base
```
Comment (qzheng527) [inline r-base/README.md]: What is this command for? "-d" means running in background.

Comment (shaoyuta) [reply r-base/README.md]: Yes, you are right. Can't use "-d" here, I have updated the patch again.
Thanks.

**Pull request merged**